src/NUnitEngine/nunit.engine.core/Internal/RuntimeFrameworks/DotNetFrameworkLocator.cs

```diff
@@ -77,6 +77,33 @@ private static IEnumerable<RuntimeFramework> FindExtremelyOldDotNetFrameworkVers
                 yield return new RuntimeFramework(RuntimeType.Net, new Version("1.0." + build));
         }
 
+        private struct MinimumRelease
+        {
+            public readonly int Release;
+            public readonly Version Version;
+
+            public MinimumRelease(int release, Version version)
+            {
+                Release = release;
+                Version = version;
+            }
+        }
+
+        private static readonly MinimumRelease[] ReleaseTable = new MinimumRelease[]
+        {
+            // TODO: Make 3-component versions work correctly
```

Comment:
Can you elaborate on this comment, please? What isn't working correctly with them?

Reply:
Obsolete comment... removing.